test/integration/test_seq_copy_int.py

```diff
@@ -33,14 +33,22 @@
      " --decode-and-evaluate 0",
      "--beam-size 2 --softmax-temperature 0.01",
      True, False, False),
-    # "Vanilla" LSTM encoder-decoder with attention, greedy and skip topk
+# "Vanilla" LSTM encoder-decoder with attention, greedy and skip topk
     ("--encoder rnn --decoder rnn --num-layers 1 --rnn-cell-type lstm --rnn-num-hidden 8 --num-embed 4 "
      " --rnn-attention-type mlp"
      " --rnn-attention-num-hidden 8 --batch-size 2 --loss cross-entropy --optimized-metric perplexity --max-updates 2"
      " --checkpoint-frequency 2 --optimizer adam --initial-learning-rate 0.01 --batch-type sentence "
      " --decode-and-evaluate 0",
      "--beam-size 1 --softmax-temperature 0.01 --skip-topk",
      True, False, False),
+# "Vanilla" LSTM encoder-decoder with attention, higher nbest size
```

Comment:
comment doesn't seem to be aligned with the string.

Reply:
fixed